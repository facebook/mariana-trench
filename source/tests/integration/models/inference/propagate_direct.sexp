(method (private) "LFlow;.flow:(LData;LData;)LData;"
 (
  (load-param v0)
  (load-param v1)

  LINE(1)
  (return-object v1)
 )
)
