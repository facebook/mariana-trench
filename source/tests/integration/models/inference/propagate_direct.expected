{
  "metadata" :
  {
    "@generated" : true
  },
  "models" :
  [
    {
      "inline_as" : "Argument(1)",
      "method" : "LFlow;.flow:(LData;LData;)LData;",
      "position" :
      {
        "line" : 0,
        "path" : "Flow.java"
      },
      "propagation" :
      [
        {
          "input" : "Argument(1)",
          "output" : "Return"
        }
      ]
    }
  ]
}
